### DISCUSSION TITLE
dont download artic x heres why

### DISCUSSION THREAD

**Discussion by jaceru** [community/community#123721] [Programming Help]
### Select Topic Area

Question

### Body

so before try this executor i ran it through a quick test and i found trojan's so yeah all i was tryna say is dont download artic x

**Comment by LiteBrite82** [comment]
Thanks for posting in the GitHub Community, @jaceru ! 

We’ve moved your post to our Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.
